### DISCUSSION TITLE
Adding more use cases to Crimestones/Sunstones

### DISCUSSION THREAD

**Discussion by nguyenc117** [sunflower-land/sunflower-land#3367] [Ideas]
Now that we have Crimestones/Sunstones as the next tier resources for long-time players, I believe it is time we solved one of the game biggest issues - being too click-intensive. We could create items like Crimestone Axe or Crimestone Pickaxe which are wearables and equipping it allows user to collect resources in just one click. The integration of multiple-bumpkin also supports this idea. While these items improve gaming experience for long-time players a little bit, as it does not directly add more value to the players, the price should be low!
What are your thoughts? please give comment below

**Comment by brian-man** [comment]
I'm glad to see I'm not the only one that keeps typing "crim**e**stone". ;)

I think they have lots of plans for getting people to spend crimstone and sunstone.

The "3 taps for trees and minerals" is definitely something I'd love to have a boost or perk to bypass.  Ditto for dozens of "chest tests" when planting Sunflowers.
